packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -83,9 +83,12 @@ function hasExpectedAssetData(
 export class QuoteRequestor {
     private readonly _rfqtMakerEndpoints: string[];
     private readonly _schemaValidator: SchemaValidator = new SchemaValidator();
+    private readonly _warningLogger: (s: string) => void;
 
-    constructor(rfqtMakerEndpoints: string[]) {
+    constructor(rfqtMakerEndpoints: string[], logger: (s: string) => void = logUtils.warn.bind(logUtils)) {
         this._rfqtMakerEndpoints = rfqtMakerEndpoints;
+        this._warningLogger = logger;
+        this._warningLogger('inside QuoteRequestor constructor');
```

Comment:
do we actually want to log this line in prod?

Reply:
thank you very much, removed in https://github.com/0xProject/0x-monorepo/pull/2562/commits/141c30b173881f1f09307d951d6d8a63483f732e